docs/modules/ROOT/pages/index.adoc

```diff
@@ -61,82 +61,53 @@ WARNING: Make sure the tag matches the target release.
 
 == Basic usage
 
-This is how it looks to build an account contract using the xref:accounts.adoc[account module].
+This is how it looks to build an account contract using the xref:accounts.adoc[account] component.
 Copy the code into `src/lib.cairo`.
 
 [,javascript]
 ----
 #[starknet::contract]
 mod MyAccount {
-    use openzeppelin::account::Account;
-    use openzeppelin::account::account::PublicKeyTrait;
-    use openzeppelin::account::interface;
-    use openzeppelin::introspection::interface::ISRC5;
-    use starknet::account::Call;
-
-    // Storage members used by this contract are defined in each imported
-    // module whose `unsafe_state` is used. This design will be improved
-    // with the addition of components in the future.
-    #[storage]
-    struct Storage {}
+    use openzeppelin::account::Account as account_component;
+    use openzeppelin::introspection::src5::SRC5 as src5_component;
 
-    #[constructor]
-    fn constructor(ref self: ContractState, public_key: felt252) {
-        let mut unsafe_state = _unsafe_state();
-        Account::InternalImpl::initializer(ref unsafe_state, public_key);
-    }
+    component!(path: account_component, storage: account, event: AccountEvent);
+    component!(path: src5_component, storage: src5, event: SRC5Event);
```

Comment:
should each of these be next of their impls?

Reply:
I think is easier to read with the component! macro calls on top, because you can quickly get what components are being used. I don't think there's much value on moving them next to the impls, but no strong opinion.

Reply:
agree on making components easy to understand at a quick glance. Just to counterbalance, the value I see is grouping logically related lines, making it easy to:

- add/remove components without affecting all lines
- make sure all directives for a component are in place (event, component, importing impl)
- perform reviews simpler and easier, based on the above items